src/Styleguide/Components/ArtistCard.tsx

```diff
@@ -45,18 +42,32 @@ export const LargeArtistCard = props => (
   </BorderBox>
 )
 
-export const SmallArtistCard = props => (
+export const SmallArtistCard = (props: ArtistCard_artist) => (
   <BorderBox hover width="100%" justifyContent="space-between">
     <Flex flexDirection="column" justifyContent="center">
-      <Sans weight="medium" size="2">
-        {props.badge}
-      </Sans>
-      <Serif size="3t">{props.headline}</Serif>
-      <Sans size="1">{props.subHeadline}</Sans>
+      <Serif size="3t">{props.name}</Serif>
+      <Sans size="1">{props.formatted_nationality_and_birthday}</Sans>
       <Button size="small" variant="secondaryOutline" width="70px" mt={1}>
         Follow
       </Button>
     </Flex>
-    <Avatar size="small" src={props.src} ml={2} />
+    {props.image && (
+      <Avatar size="small" src={props.image.cropped.url} ml={2} />
+    )}
   </BorderBox>
 )
+
+export const ArtistCardFragmentContainer = createFragmentContainer(
+  ArtistCard,
+  graphql`
+    fragment ArtistCard_artist on Artist {
```

Comment:
`ArtistCard_artist` should be read as `[module]_[prop name]`, i.e. the container will expect an input prop called `artist` and holds the data requested from `Artist`.

Reply:
I see that now in the docs...still would be nice to have gotten that as a warning or error or something? All good now 👍 , thanks.